docs/guides/api/rest-api.mdx

```diff
@@ -714,18 +716,25 @@ public class Read {
                 var id = paths[paths.length - 1];
 
                 try {
-                    var json = new Documents().collection("orders")
-                        .doc(id, Order.class)
-                        .getJson();
+                    var order = new Documents().collection("orders")
+                        .doc(id)
+                        .get();
+
+                    var json = new ObjectMapper().writeValueAsString(order);
 
                     context.getResponse()
                         .contentType("application/json")
-                        .data(json);
+                        .text(json);
 
                 } catch (NotFoundException nfe) {
                     context.getResponse()
                         .status(404)
-                        .data("Document not found: " + id);
+                        .text("Document not found: %s", id);
+
+                } catch (IOException ioe) {
+                    context.getResponse()
+                        .status(500)
+                        .text("Error getting order: %s", ioe);
```

Comment:
Are these changes related to the core of this PR, happy to go ahead with this. But I think it would be good to start avoiding tag along changes in future, and raise additional or unrelated changes as separate PRs (helps keep reviews more focused).

Reply:
The Java SDK changes predated, this PR but I used the opporunity to update these. Ideally these need to be code snipets. I will raise PR against the SDKs to migrate these across.